snuba/optimize.py

```diff
@@ -112,4 +117,6 @@ def optimize_partitions(
 
         query = (query_template % args).strip()
         logger.info(f"Optimizing partition: {part.name}")
+        start = time.time()
         clickhouse.execute(query)
+        metrics.timing("optimized_part", time.time() - start, tags={"table": table})
```

Comment:
is there any value in tagging the partition as well? I'm not sure if that's needed or not

Reply:
we cannot tag partitions. Their names are dates. Which means we will be increasing the tags over time.
You can tag partition as the number of weeks from today.